objects_test.go

```diff
@@ -9,6 +9,42 @@ import (
 
 var _ = Describe("Objects", func() {
 
+	Context("Grid object", func() {
+
+		grid := Grid{}
+		gridJSON := `{}`
```

Comment:
Testing objects with no data is not sufficient - especially unmarshalling.

Reply:
Done